tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -33,16 +33,16 @@ ${mntNamed}=  /mnt/named
 
 
 *** Keywords ***
-Setup ESX And NFS Suite
+Setup VC And NFS Suite
     [Timeout]    110 minutes
     Run Keyword And Ignore Error  Nimbus Cleanup  ${list}  ${false}
     Log To Console  \nStarting test...
 
-    ${nfs}  ${nfs_ro}  ${esx1}  ${nfs_ip}  ${nfs_ro_ip}  ${esx1_ip}=  Deploy Simple NFS Testbed  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  additional-args=--testbedSpecRubyFile /dbc/w3-dbc302/rashok/vic-nfs.rb --esxBuild ${ESX_VERSION} --esxPxeDir ${ESX_VERSION} --plugin testng
+    ${nfs}  ${nfs_ro}  ${vc}  ${nfs_ip}  ${nfs_ro_ip}  ${vc_ip}=  Deploy Simple VC Cluster With NFS Testbed  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  additional-args=--testbedSpecRubyFile /dbc/sc-dbc1215/rogelios/vic-simple-cluster-with-multiple-nfs.rb --vcvaBuild ${VC_VERSION} --esxBuild ${ESX_VERSION} --esxPxeDir ${ESX_VERSION} --plugin testng
 
-    Set Suite Variable  @{list}  ${esx1}  ${nfs}  ${nfs_ro}
-    Set Suite Variable  ${ESX1}  ${esx1}
-    Set Suite Variable  ${ESX1_IP}  ${esx1_ip}
+    Set Suite Variable  @{list}  ${vc}  ${nfs}  ${nfs_ro}
```

Comment:
you need to set all the esx here as well, otherwise they will leak

Reply:
Good catch, I'll add that in.